datalad/core/distributed/clone.py

```diff
@@ -60,8 +60,6 @@
     _test_existing_clone_target,
     _try_clone_candidates,
     decode_source_spec,
-    # RIA imports needed b/c datalad-next imports it from here ATM;
-    # Remove after core was released and next dropped the ria patch.
     postclone_preannex_cfg_ria,
     postclonecfg_ria,
 )
```

Comment:
@bpinsard do you know the way to keep this comment close to those imports? we could may be make it a separate import statement and ignore it for isort with `# isort:skip`?